site/source/tutorials/006_deploying_to_production/index.md

```diff
@@ -0,0 +1,143 @@
+
+{% aside 'Arguments vs .dojorc' %}
+It is possible to provide different settings when using command line arguments and `.dojorc`. When this occurs, command line arguments always take precedence over configuration settings stored in `.dojorc`.
+{% endaside %}
+
+The `.dojorc` file contains a JSON object with configuration information for any of the commands that are run via the `dojo` command line tool, such as `dojo build`. Each command is allocated a section of the file where its settings can be stored.
+
+Consider the following:
+
+```json
+{
+  "build-webpack": {
+    "locale": "en",
+    "messageBundles: [ "src/nls/main" ]
+  }
+}
+```
+
+This is a sample `.dojorc` file. This example contains configuration settings for the `dojo build` command. Note the field name, `build-webpack`. This is the full name of the command that we have been using - `dojo build webpack` is the default sub-command when using the `dojo build` command. That entry contains an object with two values - `locale` and `messageBundles`. Each of these fields takes values that are similar to the same options when provided by the command line.
+
+Dojo 2's build system is designed to encapsulate the build process as completely as possible. However, there may be times when a greater degree of control is required. In those situations, a project can be *ejected* from the `dojo` command line tool. We will take a look at that next.
+
+{% section %}
+
+## Taking full control
+
+{% task 'Take full control of the build process.' %}
+
+The build tool is designed to cover the most common use cases for developing and deploying Dojo 2 applications. There are times, however, when you need to take full control of the deployment process. To achieve this level of control, a project can be *ejected* from the `dojo` tool and all of its configuration information exported.
+
+**Note:** This is a *non-reversable* process. Once a project has been ejected, the `dojo` tool can no longer be used to manage a project.
+
+To eject a project, use the `dojo eject` command. You will be prompted to ensure that you understand that this is a non-reversible action. Entering **'y'** will begin to export process. The export process generates a new directory - `config` that contains all of the exported configuration information for each of the `dojo-cli` tools that the project had been using. The process will also install some additional dependencies that the project now requires.
+
+The project is now configured to be managed as a webpack project. Changes can be made to the build configuration by altering the `config/build-webpack/webpack.config.js` file. A build can then be triggered by running `webpack`'s build command and providing the configuration file. That command is: `./node_modules/.bin/webapack --config ./config/build-webpack/webpack.config.js`.
```

Comment:
command and providing the configuration file -> command and providing the configuration